### PR TITLE
2nd flywheel for shooter

### PR DESCRIPTION
**Pull request by fotnitfpro000**

### CODE REVIEW

File: components/shooter.py [hunk 1]
```diff
@@ -115,3 +128,4 @@ def execute(self) -> None:
 
         flywheel_request = VelocityVoltage(self.desired_flywheel_speed)
         self.flywheel.set_control(flywheel_request)
+        self.flywheel1.set_control(flywheel_request)
```
Comment (auscompgeek) [inline components/shooter.py:131]: One of the motor controllers should be following the other. The follow mode is more efficient.

**Comment by Kazoras** [comment]
Follow has a parameter for inversion. Also pass in the Talon ID rather than the object

**Comment by fotnitfpro000** [comment]
do i need to set the inversion in the flywheel motor configs when i set it in the follow parameter

File: components/shooter.py [hunk 2]
```diff
@@ -135,3 +146,4 @@ def execute(self) -> None:
 
         flywheel_request = VelocityVoltage(self.desired_flywheel_speed)
         self.flywheel.set_control(flywheel_request)
+        self.flywheel1 = Follower(TalonIds.shooter_flywheel, True)
```
Comment (outsidermm) [inline components/shooter.py:149]: Looking at the examples on [https://v6.docs.ctr-electronics.com/en/latest/docs/migration/migration-guide/control-requests-guide.html](url)
It looks like you can set flywheel1 control request with Follower but can’t assign Follower with TalonFX.
`self.flywheel1.set_control(Follower(…))`

File: components/shooter.py [hunk 3]
```diff
@@ -73,6 +79,7 @@ def __init__(self) -> None:
         flywheel_config.apply(flywheel_motor_config)
         flywheel_config.apply(flywheel_pid)
         flywheel_config.apply(flywheel_gear_ratio)
+        flywheel_config.apply(flywheel1_motor_config)
```
Comment (auscompgeek) [inline components/shooter.py:80]: We shouldn't set multiple of the same config type.
```suggestion
```

File: components/shooter.py [hunk 4]
```diff
@@ -71,6 +77,7 @@ def __init__(self) -> None:
         flywheel_config.apply(flywheel_motor_config)
         flywheel_config.apply(flywheel_pid)
         flywheel_config.apply(flywheel_gear_ratio)
```
Comment (auscompgeek) [inline components/shooter.py:79]: I think we probably want to set the same configs on both motors? In particular the motor output config

**Review by outsidermm — COMMENTED** [review]
-

File: components/shooter.py [hunk 5]
```diff
@@ -132,4 +142,5 @@ def execute(self) -> None:
         self.inclinator.set(inclinator_speed)
 
         flywheel_request = VelocityVoltage(self.desired_flywheel_speed)
-        self.flywheel.set_control(flywheel_request)
+        self.flywheel_left.set_control(flywheel_request)
+        self.flywheel_right.set_control(Follower(TalonIds.shooter_flywheel_left, True))
```
Comment (auscompgeek) [inline components/shooter.py:146]: A follower command should only be set once at robot start, like CTRE's example: https://github.com/CrossTheRoadElec/Phoenix6-Examples/blob/536bfad58a4c94f0e3b766aff71d763322b35a0b/python/ArcadeDrive/robot.py#L32

**Review by auscompgeek — APPROVED** [review]
LGTM. We should follow up with updating the simulation to account for the second motor.

**Pull request merged**